### DISCUSSION TITLE
Question for the Openthread Throughput

### DISCUSSION THREAD

**Discussion by eric-bj-kang** [openthread/openthread#7941] [Q&A] [answered]
Hi Team,

I'm finding a Openthread Throughput information.
In the below link, I could find the relative information.

Would you share the relative information like as below?

> Relative Sites
  - Testing, in Openthread.io
    . https://openthread.io/testing
  - Average Throughput, in Openthread.io
    . https://openthread.io/testing/performance-dashboard/average-throughput

> Average Throughput Test Information
  - Test Environment
  - Test Method
  - Test Code for nRF52840

Best Regards,
Eric.

**Comment by jwhui** [comment, marked as answer]
See the [OpenThread Quality Dashboard white paper](https://openthread.io/static/testing/OpenThread_Quality_Dashboard.pdf) for information about test environment, test method, and source code.
